generate_report.py

```diff
@@ -0,0 +1,46 @@
+from drift import info_parser
+from drift.app import create_app
+import json
+
+from kerlescan.inventory_service_interface import interleave_systems_and_profiles
+
+print("Testing report generation...")
+# TODO: remove need to initialize flask app
+app = create_app()
+app.app.app_context().push()
+
+systems, profiles, baselines, hsps = [], [], [], []
+with open("tests/reports/inputs/inventory/report1-systems.json", "r") as myfile:
+    systems = json.loads(myfile.read())["results"]
+
+with open("tests/reports/inputs/inventory/report1-system_profiles.json", "r") as myfile:
```

Comment:
Can you just use the same name (myfile) over and over while it's still open from the previous file like this?

Reply:
it closes once it leaves the "with" statement due to python magic :mage_woman: . Basically "with" will create a context where the file is open, and automatically close it once we leave the code block.

If I didn't use the with statement, I'd have to do something like this:

```python
try:
  myfile = open("tests/reports/inputs/inventory/report1-systems.json", "r")
  systems = json.loads(myfile.read())["results"]
except:
  # handle exceptions here
finally:
  myfile.close()

https://docs.python.org/3.8/reference/compound_stmts.html#with

Reply:
I cleaned this up, marking as resolved